### PR TITLE
ENH: adding prepare_test_55 method

### PR DESCRIPTION
**Pull request by nilswagner**

### CODE REVIEW

File: pyuff.py [hunk 1]
```diff
@@ -763,6 +763,8 @@ def _write55(self, fh, dset):
             #             if dset.has_key('r4') and dset.has_key('r5') and dset.has_key('r6'):
             if ('r4' in dset) and ('r5' in dset) and ('r6' in dset):
                 nDataPerNode = 6
+            else:
+                nDataPerNode = 3
```
Comment (jankoslavic) [inline pyuff.py]: @nilswagner Why is this else required?

Comment (nilswagner) [reply pyuff.py]: If you have translational DOFs solely (i.e. no rotational DOFs).

Comment (nilswagner) [reply pyuff.py]: if the two lines are missing ...

>>> import pyuff
>>> pyuff.prepare_test_55('nils.unv')
Traceback (most recent call last):
  File "/home/wagner/git/pyuff/pyuff.py", line 780, in _write55
    dset['spec_data_type'], dataType, nDataPerNode))
UnboundLocalError: local variable 'nDataPerNode' referenced before assignment

**Comment by nilswagner** [comment]
@jankoslavic Please can you help me with the test failure.

**Comment by jankoslavic** [comment]
This issue should now be resolved with the version 1.22

**Pull request closed without merging**